src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -86,5 +86,4 @@ public Key(T defaultValue) {
        Object value = store.get(key);
        return value != null ? (T)value : key.defaultValue;
```

Comment:
While we are at doing style adjustments: I usually prefer to put a blank in between the type and the `value` that should be casted. Feel free to add a commit to fix that as well :wink: